icepyx/core/query.py

```diff
@@ -939,3 +940,34 @@ def visualize_spatial_extent(
             world.plot(ax=ax, facecolor="lightgray", edgecolor="gray")
             gdf.plot(ax=ax, color="#FF8C00", alpha=0.7)
             plt.show()
+
+    def visualize_elevation(self):
+
+        product = self.dataset
+        if self.extent_type == "bounding_box":
+            bbox = self._spat_extent
+
+        else:
+            mrc_bound = self._spat_extent.minimum_rotated_rectangle
+            # generate bounding box
+            lonmin = min(mrc_bound.exterior.coords.xy[0])
+            lonmax = max(mrc_bound.exterior.coords.xy[0])
+            latmin = min(mrc_bound.exterior.coords.xy[1])
+            latmax = max(mrc_bound.exterior.coords.xy[1])
+
+            bbox = [lonmin, latmin, lonmax, latmax]
+
+        date_range = [self._start.strftime('%Y-%m-%d'),
+                      self._end.strftime('%Y-%m-%d')] if hasattr(self, '_start') else None
+
+        cycles = self._cycles if hasattr(self, '_cycles') else None
+
+        tracks = self._tracks if hasattr(self, '_tracks') else None
+
+        print(product, bbox, date_range, cycles, tracks)
```

Comment:
Is there a purpose for showing this to the user, or can we remove it once we're done development?

Reply:
Yeah we can remove it when we finishing this code, sorry it's for testing yesterday! I also need to update the test script, previously it just tested whether the returned `xarray.dataset` has data or not.

Reply:
No worries! My code always has a lot of print statements while I'm writing it. : )